mesh/pmesh.cpp

```diff
@@ -2478,6 +2488,8 @@ GetSharedFaceTransformations(int sf, bool fill2)
       GetGhostFaceTransformation(&FaceElemTr, face_type, face_geom);
    }
 
+   FaceElemTr.SetConfigurationMask(mask);
```

Comment:
The logic for the mask can be simplified to just:
```c++
   FaceElemTr.SetConfigurationMask(fill2 ? 31 : 21);
```
If you prefer to keep the current version, note that `mask += 16` is missing in the case when `is_ghost == true`.

Reply:
Thanks, I missed that.

I do prefer to keep these separate so that we acknowledge that this is a mask and we might want to use that fact in this method one day.  If we just set it in one place the reader would need to go elsewhere to lookup the meaning of the numbers 31 vs. 21.